### ISSUE TITLE
Broken support for latin-extended?

### ISSUE THREAD

**Issue by alkamid** [mmistakes/minimal-mistakes#170]
Have a look at this post: http://alkamid.com/open-data/

Accented letters are conspicously larger than the standard latin letters:

![screenshot] (http://i.imgur.com/88PYnKy.png)

I am using PT Serif, which is included in _sass/vendor/google/fonts.scss. I even replaced (locally) these with `@font-face`s found on http://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700&subset=latin-ext,latin, but it didn't help. Still, I might have had a broken local font, so I deleted `local` entries from those, but it didn't change anything. I am on Ubuntu 14.04. I also checked on Virtualbox Windows XP and on another computer with Ubuntu 12.04 — on XP it looks even much worse. Then I checked on Chrome on an Android phone and the fonts there look just fine.

Can you replicate this in your browser? If yes/no, what might be the reason?

**Comment by mmistakes** [comment]
If I remember correctly I'm not using the CSS from `_sass/vendor/google/fonts.scss` and instead am loading PT Serif directly from Google using their provided script. If you look at `assets/css/main.scss` you'll see `_font.scss` is commented out.

That could be why you didn't see any difference when you switched to the extended latin font subset. Try changing it in [`includes/_head.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/_head.html#L38) instead and use the `<script>` code Google provides.

But before you go that far I would test the text on [Google Fonts](https://www.google.com/fonts) to see if PT Serif displays correctly with another subset. There is a box you can enter whatever text you want to preview the font with.

![fonts](https://cloud.githubusercontent.com/assets/1376749/9224703/8a095b76-40d2-11e5-9ca8-ba3eaa2899c8.jpg)

If it renders the glyphs poorly there too, then you'll have to find an alternate font to use. Another option is not to use any webfonts and just stick with a system font like Georgia. Might not look as nice but you'll get the widest possible support for all characters and glyphs.

Hope this helps.

**Comment by alkamid** [comment]
Thanks for pointing me to the right direction. I should have mentioned before that I had checked PT Serif on Google Fonts and it did render the glyphs properly.

I followed your advice and added

```
<link href="http://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700|PT+Sans:400,700,400italic|PT+Sans+Narrow:400,700&subset=latin-ext,latin" rel='stylesheet' type='text/css'>
```

to [`includes/_head.html`](_includes/_head.html). The fonts are now nicely rendered everywhere (Firefox on my Ubuntu 14.04, Firefox & Chrome on XP), but the issue persists on my Chrome @ Ubuntu (yes, I cleared the cache). I know it is most likely a problem with my system, but I'd be interested in its cause, as other people (i.e. readers) might experience the same issue.
